### PR TITLE
[conv-diff] Add new variable TRANSFER_COEFFICIENT

### PR DESCRIPTION
**Pull request by riccardotosi**
Add new variable TRANSFER_COEFFICIENT, register as default in transfer_coefficient_variable and update Project Parameters of MultilevelMonteCarloApplication.

### CODE REVIEW

File: applications/convection_diffusion_application/convection_diffusion_application_variables.h [hunk 1]
```diff
@@ -35,6 +35,7 @@ KRATOS_DEFINE_APPLICATION_VARIABLE( CONVECTION_DIFFUSION_APPLICATION, double, ME
 KRATOS_DEFINE_APPLICATION_VARIABLE( CONVECTION_DIFFUSION_APPLICATION, double, MELT_TEMPERATURE_2)
 KRATOS_DEFINE_APPLICATION_VARIABLE( CONVECTION_DIFFUSION_APPLICATION, double, PROJECTED_SCALAR1)
 KRATOS_DEFINE_APPLICATION_VARIABLE( CONVECTION_DIFFUSION_APPLICATION, double, THETA)
+KRATOS_DEFINE_APPLICATION_VARIABLE( CONVECTION_DIFFUSION_APPLICATION, double, TRANSFER_COEFFICIENT)
```
Comment (philbucher) [inline applications/convection_diffusion_application/convection_diffusion_application_variables.h:38]: +1 for defining variables in applications

**Review by rubenzorrilla — COMMENTED** [review]
Why don't you take the chance to add the variable to Python?

File: applications/MultilevelMonteCarloApplication/tests/PoissonSquareTest/parameters_poisson_coarse.json [hunk 2]
```diff
@@ -7,67 +7,20 @@
...
         "end_time"        : 1.0,
-        "echo_level"      : 1,
-	"number_samples"  : 2,
-	"stochastic_pdf"  : {
-			"beta_distribution"      : {
-				"alpha"       : 2.0,
-				"beta"	      : 6.0
-			}
-	}
+        "echo_level"      : 1
```
Comment (rubenzorrilla) [inline applications/MultilevelMonteCarloApplication/tests/PoissonSquareTest/parameters_poisson_coarse.json]: Do you really want such level of verbosity in the tests?

Comment (riccardotosi) [reply applications/MultilevelMonteCarloApplication/tests/PoissonSquareTest/parameters_poisson_coarse.json]: No I don't, I haven't noticed, thanks for the hint!

**Comment by riccardotosi** [comment]
@rubenzorrilla done

**Review by rubenzorrilla — APPROVED** [review]
I'm OK with the changes but please run the convection_diffusion_application tests before merging.

**Comment by riccardotosi** [comment]
Done

**Comment by philbucher** [comment]
@riccardotosi in general you should always run the tests in the application where you modify some things

Additionally if you modify things in the Core then you should run all the tests (kratos/python_scripts/run_tests.py)
=> you could do this even all the time, then you could also report failing tests, such that the corresponding devs can fix them :+1:

**Comment by riccardotosi** [comment]
Thanks for the hints @philbucher!

**Pull request merged**